op-node/rollup/derive/channel_test.go

```diff
@@ -99,3 +106,60 @@ func TestFrameValidity(t *testing.T) {
 		t.Run(tc.name, tc.Run)
 	}
 }
+
+func TestBatchReader(t *testing.T) {
+	// Get batch data
+	rng := rand.New(rand.NewSource(0x543331))
+	singularBatch := RandomSingularBatch(rng, 5000, big.NewInt(333))
```

Comment:
```suggestion
	singularBatch := RandomSingularBatch(rng, 20, big.NewInt(333))
```
I think a smaller value like 20 gives us the same test value and makes for a quicker test.